src/material-experimental/mdc-form-field/directives/error.ts

```diff
@@ -22,11 +22,25 @@ export const MAT_ERROR = new InjectionToken<MatError>('MatError');
   selector: 'mat-error',
   host: {
     'class': 'mat-mdc-form-field-error mat-mdc-form-field-bottom-align',
-    'role': 'alert',
+    'aria-atomic': 'true',
     '[id]': 'id',
   },
   providers: [{provide: MAT_ERROR, useExisting: MatError}],
 })
-export class MatError {
+export class MatError implements AfterViewInit {
   @Input() id: string = `mat-mdc-error-${nextUniqueId++}`;
+
+  /**
+   * Aria live value of the error. By default we set this to 'polite', instead of using role='alert',
+   * so that the screen readers do not interrupt the current task to read this aloud.
+   */
+  @Input('aria-live') ariaLive: string = this._ariaLive ?? 'polite';
+
+  constructor(@Attribute('aria-live') private _ariaLive: string, private _elementRef: ElementRef){}
```

Comment:
Looking at this again, I think it might be more straightforward to do something like this
```ts
constructor(@Attribute('aria-live') ariaLive: string, private _elementRef: ElementRef) {
  if (!ariaLive) {
    _elementRef.nativeElement.setAttribute('aria-live', 'polite');
  }
}
```

Essentially we set the `aria-live` attribute if one isn't set, but literally any user-provided value will win if specified via binding. This way we don't have to introduce a new `ariaLive` input, and consumers can continue using the standard `[atter.aria-live]` binding if they want to change it.

Reply:
Agreed, this would certainly simplify things -- I've just been preserving the Input() per the suggestion that it causes it to be added to the documentation [1] but I don't have any strong preference and am more than happy to remove it.

[1] https://github.com/angular/components/pull/21870#discussion_r574841485

Reply:
Went ahead and just implemented this change.